Postman/dex.postman_collection.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
postman file tests dont work for me, I suggest making this another issue, since it's semi-complicated to fix and is holding up a working feature because of tech issues at the postman side.

Reply:
There was an issue with the variable. Discussed with Max in Slack and it works on his environment aswell nog.